src/UserService.Business/Commands/Certificate/EditCertificateCommand.cs

```diff
@@ -23,111 +25,113 @@
+      Guid? imageId = null;
+
+      if (addImageRequest == null)
+      {
+        return null;
+      }
+
+      Guid userId = _httpContextAccessor.HttpContext.GetUserId();
+
+      const string errorMessage = "Can not add certificate image to certificate. Please try again later.";
+
+      try
+      {
+        Response<IOperationResult<Guid>> response = _rcImage.GetResponse<IOperationResult<Guid>>(
+          ICreateImagesRequest.CreateObj(_createImageDataMapper.Map(
+            new List<AddImageRequest>() { addImageRequest }, userId),ImageSource.User)).Result;
+
+        if (!response.Message.IsSuccess)
         {
-            Guid? imageId = null;
-
-            if (addImageRequest == null)
-            {
-                return null;
-            }
-
-            Guid userId = _httpContextAccessor.HttpContext.GetUserId();
-
-            const string errorMessage = "Can not add certificate image to certificate. Please try again later.";
-
-            try
-            {
-                var response = _rcImage.GetResponse<IOperationResult<Guid>>(
-                    IAddImageRequest.CreateObj(
-                        addImageRequest.Name,
-                        addImageRequest.Content,
-                        addImageRequest.Extension,
-                        userId)).Result;
-
-                if (!response.Message.IsSuccess)
-                {
-                    _logger.LogWarning("Can not add certificate image to certificate. Reason: '{Errors}'",
-                        string.Join(',', response.Message.Errors));
-
-                    errors.Add(errorMessage);
-                }
-                else
-                {
-                    imageId = response.Message.Body;
-                }
-            }
-            catch (Exception exc)
-            {
-                _logger.LogError(exc, errorMessage);
-
-                errors.Add(errorMessage);
-            }
-
-            return imageId;
-        }
+          _logger.LogWarning("Can not add certificate image to certificate. Reason: '{Errors}'",
+            string.Join(',', response.Message.Errors));
 
-        public EditCertificateCommand(
-            IAccessValidator accessValidator,
-            IHttpContextAccessor httpContextAccessor,
-            IUserRepository userRepository,
-            ICertificateRepository certificateRepository,
-            IPatchDbUserCertificateMapper mapper,
-            IRequestClient<IAddImageRequest> rcImage,
-            ILogger<EditCertificateCommand> logger)
+          errors.Add(errorMessage);
+        }
+        else
         {
-            _accessValidator = accessValidator;
-            _httpContextAccessor = httpContextAccessor;
-            _userRepository = userRepository;
-            _certificateRepository = certificateRepository;
-            _mapper = mapper;
-            _rcImage = rcImage;
-            _logger = logger;
+          imageId = response.Message.Body;
```

Comment:
return body

Reply:
will be done in https://github.com/LT-Students/DigitalOffice-UserService/pull/122